.github/workflows/main.yml

```
@@ -0,0 +1,44 @@
+name: Singularity container build
+
+on:
+  push:
+    branches: [ master,ghaction ]
+  schedule:
+    - cron: "0 4 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout bot code
+        uses: actions/checkout@v2
+        with:
+          repository:  BioContainers/singularity-build-bot
+          ref: ghaction
```

```suggestion
          ref: master
```
commit before merging